vscode-lean4/src/utils/leanInstaller.ts

```diff
@@ -13,12 +13,14 @@ export class LeanInstaller implements Disposable {
     private subscriptions: Disposable[] = [];
     private prompting : boolean = false;
     private pkgService : LeanpkgService;
+    private defaultToolchain : string;
 
     private installChangedEmitter = new EventEmitter<string>();
     installChanged = this.installChangedEmitter.event
 
-    constructor(outputChannel: OutputChannel, localStorage : LocalStorageService, pkgService : LeanpkgService){
+    constructor(outputChannel: OutputChannel, localStorage : LocalStorageService, pkgService : LeanpkgService, defaultToolchain : string) {
```

Comment:
As far as I can tell, the `defaultToolchain` property of the installer is never actual used (and seems to conceptually conflict with the fact that a different `defaultToolchain` is passed to many of its functions).